tt_metal/hw/ckernels/blackhole/metal/llk_api/llk_math_common_api.h

```diff
@@ -21,9 +21,11 @@
 /*************************************************************************
  * LLK MATH COMMON
  *************************************************************************/
-template <bool untilize_en = false>
-inline void llk_math_hw_configure_disaggregated() {
-   _llk_math_hw_configure<untilize_en>();
+template <bool untilize_en = false, bool skip_inputs = false>
```

Comment:
For BH, I am unsure about adding this data format reconfig with `llk_math_hw_configure`, because there are 3 other configs in BH that are also configured in this call, and if someone just wants to do data format reconfigs, they would also be affecting these 3 configs ?

Can math have its own `llk_math_data_format_reconfig`?